Makefile

```diff
@@ -18,27 +18,40 @@ CPPCHECK=ls lib/*.cc khmer/_khmermodule.cc | grep -v test | cppcheck -DNDEBUG \
 	 --quiet -Ilib -Ithird-party/bzip2 -Ithird-party/zlib \
 	 -Ithird-party/smhasher
 
-all: khmer/_khmermodule.so
 
+## all                  : default task; show list of build commands
+all: commands
```

Comment:
This breaks `make` from compiling the code which is the number one expectation I have of a Makefile :-)